arangodb-net-standard.Test/ApiClientTestFixtureBase.cs

```diff
@@ -20,10 +20,14 @@ public abstract class ApiClientTestFixtureBase : IDisposable, IAsyncLifetime
 
         public string ArangoDbHost => Environment.GetEnvironmentVariable("ARANGO_HOST") ?? "localhost";
 
+        public int ArangoDbPort => int.TryParse(
+            Environment.GetEnvironmentVariable("ARANGO_PORT"),
+            out int dbPort) ? dbPort : 8529;
```

Comment:
I would have liked the ability to write the host and port values to the standard output when the tests execution starts. If someone has set an invalid port number, we silently fallback to `8529` which may not be obvious from the outside. In NUnit there is the `SetUpFixture` but I didn't find something similar in XUnit.

We could however write to the `Console` in the `ApiClientTestFixtureBase` constructor, which would then write the values whenever a test class is executed. @rossmills99 Do you think it's worth?

Reply:
No I don't think it's worth worrying about just now